pyodide-build/pyodide_build/buildall.py

```diff
@@ -271,7 +271,7 @@ def print_with_progress_line(str, progress_line) -> None:
         print(progress_line, end="\r")
 
 
-def get_progress_line(package_set) -> str | None:
+def get_progress_line(package_set: dict[str, Any]) -> str | None:
```

Comment:
```suggestion
def get_progress_line(package_set: dict[str, None]) -> str | None:
```
`package_set` is a sorted set, so we use a dictionary.